ansible/configs/rosa-consolidated/default_vars.yml

```diff
@@ -26,7 +26,6 @@ cloud_tags:
 # Bastion user
 bastion_user_name: rosa
 bastion_user_enable_sudo: false
-bastion_user_password: ""
```

Comment:
Because `bastion_user_password` is set as an ansible fact. It does not matter if it is set as empty. Also, being a default, leads to thinking a caller can pass another value, which will have not effect either.